cinema/urls.py

```diff
@@ -15,6 +15,7 @@
 router.register("cinema_halls", CinemaHallViewSet)
 router.register("movies", MovieViewSet)
 router.register("movie_sessions", MovieSessionViewSet)
+router.register("orders", OrderViewSet)
 
 urlpatterns = [path("", include(router.urls))]
```

Comment:
You can directly assign router.urls to urlpatterns in Django REST Framework. This is a concise way to define your URL patterns when using routers.
 
 urlpatterns = router.urls